include/aspect/mesh_deformation/interface.h

```diff
@@ -411,6 +442,13 @@ namespace aspect
          */
         std::set<types::boundary_id> free_surface_boundary_ids;
 
+        /**
+         * A pointer to the initial topography model.
+         */
+        InitialTopographyModel::Interface<dim> *topo_model;
```

Comment:
Rather than storing a plain pointer, is there a way how we can access this via `SimulatorAccess`?

Reply:
Yes, but this will become obsolete once Rene's initial mesh deformation functionality is merged, so then we could remove it all together.

Reply:
Changed it to going through the `SimulatorAccess` when we need the topography, so we don't need to store the pointer anymore.

Reply:
But the pointer is still in the code? Can it be removed? Also it may be a while before we can completely remove the old initial topography implementation due to backward compatibility issues.

Reply:
Yes, another left-over, unused parameter, sorry.